### PR TITLE
Fix: make member detail page responsive

### PR DESCRIPTION
**Pull request by anishpawaskar**
issue: https://github.com/Real-Dev-Squad/members-site/issues/65

**What is change?**
- Added CSS to make member details page responsive

https://github.com/Real-Dev-Squad/members-site/assets/22213872/e75d9e41-e7df-4661-83f7-4296b0b6b126

### CODE REVIEW

File: src/components/MemberContribution/ContributionAccordianItem.tsx [hunk 1]
```diff
@@ -33,31 +33,31 @@ export default function ContributionAccordianItem({
...
-      as='button'
+      as="button"
       onMouseEnter={showSetting}
       onMouseLeave={hideSetting}
       pb={4}
-      sx={{ position: 'relative' }}
+      sx={{ position: "relative", textAlign: "left", width: "100%" }}
     >
       <h3
         style={{
```
Comment (vvaibhavdesai) [inline src/components/MemberContribution/ContributionAccordianItem.tsx:43]: please remove inline styles from here

File: src/components/MemberProfileCard/memberprofile.module.css [hunk 2]
```diff
@@ -1,9 +1,13 @@
 .memberProfile_container {
-  top: 100px;
+  /* top: 100px;
   width: 18rem;
   margin-top: 8rem;
   margin-left: 6rem;
-  height: 20rem;
+  height: 20rem; */
```
Comment (vinayak-trivedi) [inline src/components/MemberProfileCard/memberprofile.module.css:6]: can we remove this commented css?

File: src/components/MembersSectionNew/components/MembersCard/index.tsx [hunk 3]
```diff
@@ -44,6 +46,11 @@ export default function MembersCard({ member }: { member: MemberType }) {
     );
   }
 
+  function routeHandler() {
+    console.log("hello", member.username);
```
Comment (vinayak-trivedi) [inline src/components/MembersSectionNew/components/MembersCard/index.tsx:50]: remove console

**Review by vinayak-trivedi — COMMENTED** [review]
please remove the commented code and console logs rest LGTM

File: src/components/MemberProfileCard/memberprofile.module.css (hunk 2 above)
Comment (anishpawaskar) [reply src/components/MemberProfileCard/memberprofile.module.css]: sure

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

File: src/components/MemberContribution/ContributionAccordian.tsx [hunk 4]
```diff
@@ -28,25 +27,35 @@ export default function ContributionAccordion({
...
-      />
+      <>
+        <ContributionAccordianItem
+          task={task}
+          key={idx}
+          title={title}
+          openTaskStatusUpdateModal={openTaskStatusUpdateModal!}
+        />
+        <hr
+          style={{
```
Comment (vvaibhavdesai) [inline src/components/MemberContribution/ContributionAccordian.tsx:54]: move this to a class

**Pull request merged**